analyzers/slither/.deepsource/issues/SLITHER-W1001.toml

```diff
@@ -0,0 +1,31 @@
+title = "Storage abiencoderv2 array"
+verbose_name = "abiencoderv2-array"
```

Comment:
Does the tool report this as `abiencoderv2-array` or `0-0-abiencoderv2-array` in sarif reports?

Reply:
The `rule.id` in Sarif reports uses `0-0-abiencoderv2-array`.

Reply:
But `abiencoderv2-array` is the unique identifier of the issue in the CLI/Wiki.

Reply:
Hmm. I was also thinking about getting rid of `issues_map.json` and just using issue tomls for this. 

Can you let me know if slither supports silencing issues via pragmas? And what does it expect there?

Reply:
Yes, it does have custom pragma's for silencing ([ref](https://github.com/crytic/slither/wiki/Usage#triage-mode)).

> By adding `//slither-disable-next-line DETECTOR_NAME` before the issue
By adding `// slither-disable-start [detector] ... // slither-disable-end [detector]` around the code to disable the detector on a large section

Reply:
wow slither. 🔮

Reply:
we still keeping issues_map then, i guess.